docker-hub/2fa/recover-hub-account.md

```diff
@@ -38,4 +38,4 @@ authentication. See [Enabling two-factor authentication on Docker Hub](/docker-h
 ## Alternative recovery methods
 
 If you have lost access to both your two-factor authentication application and
-your recovery code, send an email to [Docker Hub Support](mailto:hub-support@docker.com) from the primary email associated with your Docker ID for recovery instructions.
+your recovery code, get in touch with our Support team using the [Contact Support form](https://hub.docker.com/support/contact/?category=2fa-lockout&username=){:target="_blank" rel="noopener" class="_"}. You must enter the primary email address associated with your Docker ID in the Contact Support form for recovery instructions.
```

Comment:
We can omit the `username` property from the URL.

```suggestion
your recovery code, get in touch with our Support team using the [Contact Support form](https://hub.docker.com/support/contact/?category=2fa-lockout){:target="_blank" rel="noopener" class="_"}. You must enter the primary email address associated with your Docker ID in the Contact Support form for recovery instructions.
```

Reply:
Thanks @mark-dr. Fixed this.